contracts/SportMarkets/SportPositions/SportPositionalMarket.sol

```diff
@@ -39,13 +40,10 @@ contract SportPositionalMarket is OwnedWithInit, ISportPositionalMarket {
 
     struct SportPositionalMarketParameters {
         address owner;
-        IERC20 sUSD;
         address creator;
         bytes32 gameId;
         string gameLabel;
         uint[2] times; // [maturity, expiry]
-        uint deposit; // sUSD deposit
```

Comment:
I would leave initialMint to not have to release all related contracts, even though its not used.